types/mdarea/index.d.ts

```diff
@@ -1,132 +0,0 @@
-// Type definitions for mdarea 1.0
-// Project: https://github.com/jahudka/mdarea#readme
-// Definitions by: Antoine Laffargue <https://github.com/toniopelo>
-//                 Dan Kadera <https://github.com/jahudka>
-// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-export as namespace MarkdownArea;
-
-interface MarkdownEditorOptions {
-    /**
-     * @remarks
-     * How many spaces to use for indentation.
-     * If you specify a string, its length will be used -
-     * beware that the tab characterhas a length of 1!
-     *
-     * @default 4
-     */
```

Comment:
@jahudka You might want to copy and paste those comments in your own type files as they add documentation on mouse hover (for vscode at least, and probably other editors).
Thanks for the update @BendingBender, LGTM 👍